src/components/navigation-header/index.tsx

```diff
@@ -43,10 +46,10 @@ const NavigationHeader: React.FC = () => {
           </Link>
           <ProductSwitcher />
         </div>
-        {currentProduct?.navigationHeaderItems && (
+        {hasNavigationItems && (
           <div className={s.headerRight}>
             <ul className={s.navLinks}>
-              {currentProduct?.navigationHeaderItems.map((navLink) => {
+              {navItems.map((navLink: NavigationHeaderItem) => {
```

Comment:
There was a typedef warning on this `map` call, so that's why `NavigationHeaderItem` is now imported for use here. It made sense to also move the export of the interface to this file as well since we do that in most places.